db/builder.cc

```diff
@@ -173,12 +173,28 @@ Status BuildTable(
             prev_key.assign(keys.front());
             ok = ParseInternalKey(Slice(prev_key), &prev_ikey);
             assert(ok);
+
+            // The front item has the smallest seqno in the merge queue
```

Comment:
@siying no need to merge this, since I fixed the issue with https://reviews.facebook.net/D42087 (also took the unit test from this PR)